trustfall_stubgen/src/util.rs

```diff
@@ -138,3 +138,11 @@ pub(crate) fn field_value_to_rust_type(
         #base.#suffix
     }
 }
+
+pub fn escaped_rust_name(name: String) -> String {
+    if name == "type" {
```

Comment:
I only escape type, what other keywords should we escape?

Reply:
any keyword mentioned anywhere on this page (strict, weak, or reserved): https://doc.rust-lang.org/reference/keywords.html

Reply:
[061c32e](https://github.com/obi1kenobi/trustfall/pull/406/commits/061c32e308a1b6d5b5330524c3387bb8ac195955)